source/scenes/common/Login/Login.tsx

```diff
@@ -114,17 +120,16 @@ const Login = ({
             placeholder={PLEASE_ENTER_YOUR_PASSWORD_STRING}
             variant={styles.password}
           />
-          <div className={styles.errorWrapper}>
-            {errors.password ? (
-              <span className={errorClass}>{errors.password.message}</span>
+          <div id={'ui-login-status'} className={styles.errorWrapper}>
+            {isInvalid ? (
+              <span id={'ui-login-error'} className={errorClass}>{failureComment}</span>
             ) : (
-              isInvalid && (
-                <span className={errorClass}>Error: Invalid password</span>
-              )
+              <span id={'ui-login-success'} className={errorClass}>{successComment}</span>
             )}
           </div>
         </div>
         <ButtonV3
+          id={'ui-login-submit-button'}
```

Comment:
Please update id format to match existing style in other parts of the codebase.

Reply:
changed all ids created to follow the following pattern: "<componentName>-<functionName>"